homeassistant/components/raspberry_pi/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+  "domain": "raspberry_pi",
+  "name": "Raspberry Pi",
+  "config_flow": false,
```

Comment:
The `raspberry_pi` integration has a config flow, but it's not meant to be user initiated.
Should this be set to `true` or `false`?

Reply:
If we set it to false, the integration won't show up in the result from this view:
https://github.com/home-assistant/core/blob/692a602aeabc2f11dfed9bdb7d6ac2acd20bde8d/homeassistant/components/config/config_entries.py#L212-L224

The integration will also be excluded from the generated lists of domains that support discovery.

Reply:
Set it to false.

Reply:
I suggest we add a new integration_type that we filter out.

Reply:
That's already in the PR 👍